src/lib/code-sample-editor.ts

```diff
@@ -153,11 +175,14 @@ export class CodeSampleEditor extends LitElement {
       ${this.noFilePicker
         ? nothing
         : html` <mwc-tab-bar
+            part="file-picker"
             .activeIndex=${this._currentFileIndex ?? 0}
             @MDCTabBar:activated=${this._tabActivated}
           >
             ${this.files?.map((file) => {
-              const label = file.name.substring(file.name.lastIndexOf('/') + 1);
+              const label =
```

Comment:
Does `mwc-tab` support a tooltip where we could put the filename when a label is shown?

Reply:
I kind of feel that if you're using labels, you are typically trying to simplify things by removing the technicality of filenames (e.g. the HTML/JS example in the demo). So I'm not sure I would want to see a filename popup when I interact with the tabs -- seems like a distraction. Can discuss more later, though.